crypto_kem/kyber1024/aarch64/indcpa.c

```diff
@@ -1,15 +1,50 @@
-#include "NTT_params.h"
-#include "indcpa.h"
-#include "ntt.h"
+
+/*
+ * This file was originally licensed
+ * under Apache 2.0 (https://www.apache.org/licenses/LICENSE-2.0.html)
+ * at https://github.com/GMUCERG/PQC_NEON/blob/main/neon/kyber or
+ * public domain at https://github.com/cothan/kyber/blob/master/neon
+ *
+ * We choose
+ * CC0 1.0 Universal or the following MIT License for this file.
```

Comment:
This is also really confusing to me. Are you trying to say it's triple licensed?

Reply:
No, we license it under CC0 and MIT. The first lines say where the unmodified file comes from.

Reply:
For clarification, the first lines say where the file came from before modification. We modify and license it under CC0 and MIT.